### PR TITLE
Commenting wallaby & centos8 for 4.2.HF2

### PR DESCRIPTION
**Pull request by rajneeshkapoor**

### CODE REVIEW

File: redhat-director-scripts/docker/build_tripleo_containers.sh [hunk 1]
```diff
@@ -22,11 +22,11 @@ fi
 
 openstack_distro="tripleo"
 
-declare -a openstack_releases=("train" "wallaby")
+declare -a openstack_releases=("train")
 
-declare -a openstack_platforms=("centos7" "centos8s")
```
Comment (nasharatm) [inline redhat-director-scripts/docker/build_tripleo_containers.sh:27]: should it be "centos8" or "centos8s"

File: redhat-director-scripts/docker/publish_tripleo_containers.sh [hunk 2]
```diff
@@ -14,9 +14,9 @@ tvault_version=$1
 
 openstack_distro="tripleo"
 
-declare -a openstack_releases=("train" "wallaby")
+declare -a openstack_releases=("train")
 
-declare -a openstack_platforms=("centos7" "centos8s")
```
Comment (nasharatm) [inline redhat-director-scripts/docker/publish_tripleo_containers.sh:19]: same comment as above?

File: redhat-director-scripts/docker/build_tripleo_containers.sh (hunk 1 above)
Comment (rajneeshkapoor) [reply redhat-director-scripts/docker/build_tripleo_containers.sh:27]: it was centos8s, is removed now.

**Review by nasharatm — APPROVED** [review]

**Comment by shyam-biradar** [comment]
@rajneeshkapoor centos8s belongs to centos 8 stream and not centos8. Please let us know why we are removing it from build scripts.

**Review by shyam-biradar — APPROVED** [review]

**Pull request merged**